tests/__init__.py

```diff
@@ -1,7 +1,5 @@
-# -*- coding: utf-8 -*-
-
 """Test package for Azure IoT Edge Dev Tool ."""
 
-__author__ = 'David Ruttka'
-__email__ = 'david.ruttka@microsoft.com'
-__version__ = '0.0.1'
+__author__ = 'Microsoft Open Source'
```

Comment:
I don't think these attributes are that useful in tests unless you foresee lots of community involvement here.

Reply:
This is consistent with our other client repos (like the python sdk).